examples/layer-browser/package.json

```diff
@@ -9,8 +9,8 @@
     "start-local-production": "webpack-dev-server --env.local --env.production --progress --hot --open"
   },
   "dependencies": {
-    "@loaders.gl/ply": "^2.3.9",
-    "@loaders.gl/gltf": "^2.3.9",
+    "@loaders.gl/ply": "^3.0.0-alpha",
```

Comment:
@Pessimistress Does this work? I always seem to need to specify the actual alpha version? If it does work that is great.

Reply:
Yes that works. You'll have to specify a version if you want to bump to another alpha release later.